lib/_constants.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  CLASS_NAMESPACE: 'gmusic-theme',
+  CLASS_NAMESPACE: 'gmusic-theme.material',
```

Comment:
Sounds good.

And lol, I read the PR notes.  Came back 30 minutes later and read the diff.  Guess I should do it all at once next time :+1: